src/SpotifyApi.NetCore.Tests/PersonalizationApiTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SpotifyApi.NetCore.Authorization;
+using SpotifyApi.NetCore.Models;
+using SpotifyApi.NetCore.Helpers;
+
+namespace SpotifyApi.NetCore.Tests
+{
+    [TestClass]
+    public class PersonalizationApiTests
+    {
+        PersonalizationApi api;
+        string bearerAccessToken;
+
+        public PersonalizationApiTests()
+        {
+            var http = new HttpClient();
+            IConfiguration testConfig = TestsHelper.GetLocalConfig();
+            bearerAccessToken = testConfig.GetValue(typeof(string),
+                "SpotifyUserBearerAccessToken").ToString();
+            var accounts = new AccountsService(http, testConfig);
+            api = new PersonalizationApi(http, accounts);
+        }
```

Comment:
Reusing objects in test classes can have unintended consequences. If you want to keep your test code clean, put this code into a method called `Initialize()` with a `[TestInitialize]` attribute. This means the objects get re-initialized on every test, but you can keep your code clean.